src/shared/index.ts

```diff
@@ -143,16 +143,16 @@ export function augmentLog(log: Log, rules?: Map<string, ReportingDescriptor>) {
 
 export function effectiveLevel(result: Result): Result.level {
     switch (result.kind) {
-        case 'informational':
-        case 'notApplicable':
-        case 'pass':
-            return 'note';
-        case 'open':
-        case 'review':
-            return 'warning';
-        case 'fail':
-        default:
-            return result.level ?? result._rule?.defaultConfiguration?.level ?? 'warning';
```

Comment:
I don't agree much with this rule. What do you suggest in this case?

Reply:
I also don't agree with that rule. Just pushed a fix. Looks better?